test/test_cuda.py

```diff
@@ -2747,8 +2747,7 @@ def test_autocast_ignored_types(self):
                     self.assertTrue(torch.norm(a_ignore).dtype is type_no_autocast)
 
                 # Tests if CastPolicy::promote ops ignore double and int
-                with self.assertRaises(RuntimeError):
-                    torch.cat((a_ignore, c_16))
+                torch.cat((a_ignore, c_16))
```

Comment:
Here this cast should be valid,
I am unsure if we should just left the test but with the assertion on,
or we should just delete all these lines since now cat supports type promotion

Reply:
I'm not familiar with `torch.cuda.amp.autocast()` but if the test is no longer valid it probably makes sense to remove it. I think your other tests in test_type_promotion cover this case anyway?

Reply:
Yes, I think so too. I remove it.